idd/Energy+.idd.in

```diff
@@ -80385,6 +80389,121 @@ FaultModel:EnthalpySensorOffset:ReturnAir,
        \maximum< 20000
        \default 0.0
        \units J/kg
+   
+FaultModel:TemperatureSensorOffset:ChillerSupplyWater,
+   \memo This object describes fault of chiller supply water temperature sensor offset
+   \min-fields 6
+   A1, \field Name
+       \note Enter the name of the fault
+       \required-field
+       \type alpha
+   A2, \field Availability Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+   A3, \field Severity Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+   A4, \field Chiller Object Type
+       \note Enter the type of a chiller object
+       \required-field
+       \type choice
+       \key Chiller:Electric
+       \key Chiller:Electric:EIR
+       \key Chiller:Electric:ReformulatedEIR
+       \key Chiller:ConstantCOP
+       \key Chiller:EngineDriven
+       \key Chiller:CombustionTurbine
+       \key Chiller:Absorption
+       \key Chiller:Absorption:Indirect
+   A5, \field Chiller Object Name
+       \note Enter the name of a chiller object
+       \required-field
+       \type object-list
+       \object-list Chillers
+   N1; \field Reference Sensor Offset
+       \type real
+       \minimum> -10
+       \maximum< 10
+       \default 0.0
+       \units deltaC
+
+FaultModel:TemperatureSensorOffset:CoilSupplyAir,
+   \memo This object describes fault of coil supply air temperature sensor offset
+   \min-fields 6
+   A1, \field Name
+       \note Enter the name of the fault
+       \required-field
+       \type alpha
+   A2, \field Availability Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+   A3, \field Severity Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+   A4, \field Coil Object Type
+       \note Enter the type of the coil affected
+       \required-field
+       \type choice
+       \key Coil:Heating:Electric
+       \key Coil:Heating:Gas
+       \key Coil:Heating:Desuperheater
+       \key Coil:Heating:Steam
+       \key Coil:Heating:Water
+       \key Coil:Cooling:Water
+       \key Coil:Cooling:Water:Detailedgeometry
```

Comment:
@rongpengzhang What about CoilSystem:Cooling:DX and CoilSystem:Heating:DX?  These are both temperature-based coil controls.  Also, AirloopHVAC:UnitarySystem can serve a similar function (acting as just a coil controller with a setpoint).